src/components/modal/v2/styles/components/_offer-accordion.scss

```
@@ -22,6 +22,25 @@
             outline: 3px solid $light-blue;
             border-color: transparent;
         }
+        .v5Design {
+            button {
+                background-color: $ql-gray;
+            }
+        }
+
+        &.open {
+            &.v5Design {
+                background-color: #0070e0;
+                color: $v5-white;
+                button {
+                    background-color: #0070e0;
+                    color: $v5-white;
+                }
+                div {
+                    color: $v5-white !important;
+                }
+            }
+        }
```

We can add the below underneath your `&.open` styles to ensure the cards take up the desired width.
```suggestion
        }
        
          .v5Design & {
            width: 100%;

            &.open {
                border: 3px solid $light-blue;
            }
        }
```